.github/workflows/deploy.yaml

```diff
@@ -17,6 +17,11 @@ on:
         - staging
         - production
         default: 'integration'
+      lockIfManualDeploy:
+        description: 'Lock application to this version if manually deployed'
+        required: true
+        type: boolean
+        default: true
```

Comment:
Nit: Wondering if it'd be safer to have the default to be false. The risk with this feature is that someone manually deploys and then "forgets" to re-enable continuous deployment. If it were false then the person would to explicitly disable it, and which would act as a reminder/check that they understand that CD will be disabled.